src/test/java/InvalidLoginTests.java

```diff
@@ -0,0 +1,25 @@
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import Pages.LoginPage;
+
+public class InvalidLoginTests extends BaseTest{
+    LoginPage loginPage;
+    @BeforeMethod
+    void setup(){
+        loginPage= new LoginPage(getThreadDriver());
+        getThreadDriver().get(url);
+    }
+    @Test
+    public void invalidPassword() throws InterruptedException {
+        loginPage.provideEmail("incorrectEmail").providePassword("incotrerctPwd").clickSubmit();
+        Thread.sleep(3000);
+    }
+    @Test
+    public void emptyPassword(){
+        loginPage.provideEmail("demo@class.com").providePassword("").clickSubmit();
+    }
+    @Test
+    public void emptyEmailPassword(){
+        loginPage.provideEmail("").providePassword("").clickSubmit();
+    }
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

}
```

Please, add Assert

**One test - One Assert**

For example

![Clean_Test](https://github.com/Test-Pro-Bootcamp/A49-QA-Automation-Framework/assets/104761482/fe7aecbf-ccf5-4812-860e-789f245d3e61)